state-transition/core/core_test.go

```diff
@@ -144,7 +144,7 @@ func moveToEndOfEpoch(
 					ExtraData:    []byte("testing"),
 					Transactions: [][]byte{},
 					Withdrawals: []*engineprimitives.Withdrawal{
-						st.EVMInflationWithdrawal(),
+						st.EVMInflationWithdrawal(blk.GetSlot() + 1),
```

Comment:
@calbera question: if we do this here, should we do this in build block too?

Reply:
not needed I think

Reply:
Cross checked and I agree. In block building we do use the height passed by CometBFT for the next block to be built